test/compflags/lydia/library/exportArray/returnExternArray.chpl

```diff
@@ -1,5 +1,5 @@
-use ExternalArray;
-
+// This is a mock up of what returning an array from an exported function looks
+// like under the covers
```

Comment:
isn't it working? why the comment?

Reply:
This was a test from my initial implementation, to make sure I could use the functions I made the way I thought I could.  I decided to leave the test around, but I could see removing them now with the compiler support as a reasonable course of action.  This test would have to wait for the returning implementation, though